tests/unit/test_localesConfig.js

```diff
@@ -11,6 +11,8 @@ import { langToLocale, localeToLang } from 'core/i18n/utils';
 const langs = config.get('langs');
 const basePath = config.get('basePath');
 
+const placeholderRX = /%\(.*?\)s/g;
```

Comment:
`RX`? 🤔

Reply:
That's shorthand for Regex, would you prefer something else?

Reply:
I'm going to land this to avoid it bit-rotting due to incoming locale changes. But if you'd like this changed I can do a quick follow-up.

Reply:
yes, pattern or regex would be nice